opl/opl_sdl.c

```diff
@@ -173,8 +173,8 @@ static void FillBuffer(int16_t *buffer, unsigned int nsamples)
 
         for (i=0; i<nsamples; ++i)
         {
-            buffer[i * 2] = (int16_t) mix_buffer[i * 2];
-            buffer[i * 2 + 1] = (int16_t) mix_buffer[i * 2 + 1];
+            buffer[i * 2 + 1] = (int16_t) mix_buffer[i * 2];
```

Comment:
Alternatively, the fault may also be [in the higher level code](https://github.com/chocolate-doom/chocolate-doom/blob/master/src/i_oplmusic.c#L1136) that generates the panning values from MIDI instructions.

@khokh2001 - what do you think?